addons/mail/static/src/components/thread_viewer/thread_viewer_tests.js

```diff
@@ -355,6 +355,8 @@ QUnit.test('mark channel as fetched and seen when a new message is loaded if com
 
     const threadViewer = this.env.models['mail.thread_viewer'].create({ thread: [['link', thread]] });
     await this.createThreadViewerComponent(threadViewer, { hasComposer: true });
+
+    document.querySelector('.o_ComposerTextInput_textarea').focus();
```

Comment:
It's not weird, and it's not due to not using models here.
I'm just simulating a user action the proper way here.
It so happens that this user action is not triggering a render, because there is no interface change in the flux of focusing a composer.